.github/ISSUE_TEMPLATE.md

```diff
@@ -1,28 +1,59 @@
-
-- [CodePen with Foundation 6.3.1, Float Grid and MotionUI](http://codepen.io/IamManchanda/pen/LWGZxR)
-- [CodePen with Foundation 6.3.1, Float Grid with RTL Direction and MotionUI](http://codepen.io/IamManchanda/pen/bRYOMv)
-- [CodePen with Foundation 6.3.1, Flexbox grid and MotionUI](http://codepen.io/IamManchanda/pen/zZrBEv)
-
-- [CodePen with Foundation 6.4.1, XY Flexbox grid and MotionUI](http://codepen.io/IamManchanda/pen/EXbGKJ)
-- [CodePen with Foundation 6.4.1, XY Flexbox grid with RTL Direction and MotionUI](http://codepen.io/IamManchanda/pen/qjVLoO)
-- [CodePen with Foundation 6.4.1, XY Flexbox grid with Prototype Mode and MotionUI](http://codepen.io/IamManchanda/pen/XgzopG)
-- [CodePen with Foundation 6.4.1, Float Grid and MotionUI](http://codepen.io/IamManchanda/pen/qjVLrB)
+<!--- --------------------------------------------------------------------- -->
+<!---                 Please fill the following template                    -->
+<!---                 Your issue may be ignored otherwise                   -->
+<!--- --------------------------------------------------------------------- -->
+<!---  Only submit here bug or feature requests. For help or questions to   -->
+<!---  the community, see the forum: https://foundation.zurb.com/forum      -->
+
+## Expected Behavior
+<!--- If describing a bug, tell us what should happen.                      -->
+<!--- If suggesting a change/feature, tell us why and how it should work.   -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead.                    -->
+<!--- If suggesting a change/feature, explain the difference from           -->
+<!--- current behavior.                                                     -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug,                 -->
+<!--- or ideas how to implement the addition or change.                     -->
+
+## Test Case and/or Steps to Reproduce (for bugs)
+<!--- We highly recommand you to provide a live example of your bug so we   -->
+<!--- can reproduce it. You can create a test case with the last Foundation -->
+<!--- version by forking https://codepen.io/ncoden/pen/YLzjeq               -->
+Test Case: <!-- https://... -->
+
+<!--- If you cannot provide a test case, provide an unambiguous set of      -->
+<!--- steps to reproduce, with your code and configuration.                 -->
+How to reproduce:
+1.
+2.
+3.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish?   -->
+<!--- Providing context helps us come up with a solution that is most       -->
+<!--- useful in the real world                                              -->
+
+## Your Environment
+<!--- Include as many relevant details about the context and environment    -->
+<!--- you experienced the bug in. You can also provide logs.                -->
+- Foundation version(s) used:
+- Browser(s) Name and version(s):
```

Comment:
Name => name

What about the npm / yarn and Node.js (webpack and so on) version? Is this covered by the project link?

Reply:
I think there is only a few rare cases when we need them (like for build issue). No need to ask for them in every issue.

Reply:
You are right.